src/operators.jl

```diff
@@ -1223,23 +1223,23 @@ function resolve_convex_constr(expr::Any, m::PODNonlinearModel=nothing, idx::Int
         convex_type = :Unknown
 
         # [BUG FIX | FORK]
```

Comment:
What is this comment BUG FIX

Reply:
Let me see... I cannot replicate this exception as of now. And, apparently, I don't have a good test for this exception. It is supposed to be a safety scheme which shows up when a special type of linear constraint was provided through @NLconstraint. I will check if there exists an example from MINLPLib.

Reply:
If it does not exist - we can remove that exception